Week3/homework/3-step3.js

```diff
@@ -3,6 +3,14 @@
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
   // add your code here
+
+  if (num < 1) {
+    return '';
+  }
+  let myStr = str;
```

Comment:
Try to use variable names which actually describe what the variables are actually being used for. Names like 'mySomething' can be misleading and hence lead to confusing code
Saving credits for Hardit :heart:.